openapi/SwarmDebug.yaml

```diff
@@ -482,7 +482,7 @@ paths:
       tags:
         - Chequebook
       responses:
-        "200":
+        "201":
```

Comment:
unsure about this. @ralph-pichler is the cashout endpoint returning once the cashout is successful?

Reply:
it returns once the signed tx has been sent to the eth backend (but not confirmed yet).

Reply:
so should we keep it a 200 then?

Reply:
201 is fine in this case :+1: